cmd/scan.go

```diff
@@ -58,6 +58,26 @@ with as many details & logs as you can share!
 ` + colorstring.Redf("Error: %s", e.msg)
 }
 
+// BuildForTestingError ...
+type BuildForTestingError struct {
```

Comment:
do not export this type and its methods as you are only using this inside of the `cmd` package